include/grpcpp/ext/server_metric_recorder.h

```diff
@@ -43,9 +43,10 @@ class ServerMetricRecorder {
  public:
   // Factory method. Use this to create.
   static std::unique_ptr<ServerMetricRecorder> Create();
-  /// Records the server CPU utilization in the range [0, 1].
-  /// Values outside of the valid range are rejected.
-  /// Overrides the stored value when called again with a valid value.
+  /// Records the server CPU utilization in the range [0, infy).
+  /// Values may be larger than 1.0 when the usage exceeds the reporter
+  /// dependent notion of soft limits. Values outside of the valid range are
+  /// rejected. Overrides the stored value when called again with a valid value.
   void SetCpuUtilization(double value);
```

Comment:
Please also update the comment on `CallMetricRecorder::RecordCpuUtilizationMetric()`:

https://github.com/grpc/grpc/blob/1267000bbb690353c7d5e05b32adcffdbbeed16c/include/grpcpp/ext/call_metric_recorder.h#L43

Reply:
Done.